src/components/Button/Button.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import renderer from "react-test-renderer";
+import Button from "./Button";
+
+describe("Button", () => {
+  it("render without crashing", () => {
+    const div = document.createElement("div");
+    ReactDOM.render(<Button />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+  it("should render elements in `children` props", () => {
```

Comment:
I'm glad that you write unit tests but I think `Button` component can have more tests for example if function which we pass in `onClick` prop is called or type of button will be changed if we pass other (valid or invalid) `type` props.